crrExistingObjects.js

```diff
@@ -223,13 +243,43 @@ function _listObjectVersions(bucket, VersionIdMarker, KeyMarker, cb) {
         Prefix: TARGET_PREFIX,
         VersionIdMarker,
         KeyMarker,
-    }, cb);
+    }, (err, data) => {
+        if (err) {
+            // retry only on Network errors and if the process is set to retry
+            // on errors.
+            if (err.code === 'NetworkingError' && RETRY_ON_ERROR) {
+                log.error('Error listing object versions, retrying request',
+                    { error: err, bucketName: bucket });
+                return _listObjectVersions(bucket, VersionIdMarker,
+                    KeyMarker, cb);
+            }
+            return cb(err);
+        }
+        return cb(null, data);
+    });
+}
+
+// get bucket replication info
+function _getBucketReplication(options, cb) {
+    return s3.getBucketReplication(options, (err, res) => {
+        if (err) {
+            // retry only on Network errors and if the process is set to retry
+            // on errors.
+            if (err.code === 'NetworkingError' && RETRY_ON_ERROR) {
+                log.error('Error getting replication info, retrying request',
+                    { error: err });
+                return _getBucketReplication(options, cb);
+            }
+            return cb(err);
```

Comment:
This is the error you want to log as error.

Reply:
Yes, the callback is logging it as error.